Zotero/Scenes/Detail/Lookup/Views/LookupViewController.swift

```diff
@@ -179,11 +179,7 @@ class LookupViewController: UIViewController {
             guard let `self` = self, let value = change.newValue, value.height != self.tableViewHeight.constant else { return }
 
             self.tableViewHeight.constant = value.height
-
```

Comment:
@michalrentka with the table view height being updated here, I don't believe this condition would be met hence why I removed it.